### DISCUSSION TITLE
Unable install distro

### DISCUSSION THREAD

**Discussion by YourWifeCallYou** [microsoft/WSL#11774] [Q&A] [answered]
```
C:\WINDOWS\System32>wsl -v
WSL version: 2.2.4.0
Kernel version: 5.15.153.1-2
WSLg version: 1.0.61
MSRDC version: 1.2.5326
Direct3D version: 1.611.1-81528511
DXCore version: 10.0.26091.1-240325-1447.ge-release
Windows version: 10.0.19045.4529
```
I have a colleague who experiencing issue with installing any distro within WSL, while everyone else did not experience such issue.
We have Microsoft Store disabled; hence we could only install it via command line.

wsl --list --online is able to list the valid distro that could be installed, and Ubuntu 24.04 LTS is on the list for him.
 
Steps to reproduce the issue:

```
PS C:\WINDOWS\System32> wsl --install -d Ubuntu-24.04
Installing: Ubuntu 24.04 LTS
Error: 0x8024500c 0.0% ]
Error code: Wsl/InstallDistro/0x8024500c
```
Troubleshooting attempted: 
1. Restart PC
2. Tried with VPN gateway for different region
3. Fully re-installed WSL

Any help/suggestion is much appreciated.

**Comment by jon-freed** [comment]
Relevant:  #8603

**Comment by OneBlue** [comment, marked as answer]
@YourWifeCallYou: You're using a pretty old version of WSL. Can you try to run `wsl --update`, and see if that solves the issue ?

**Reply by rtaylor02** [reply]
> @YourWifeCallYou: You're using a pretty old version of WSL. Can you try to run `wsl --update`, and see if that solves the issue ?

This works for me - thanks!
